iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -62,8 +62,22 @@ PortManager::PortManager(RouDiMemoryInterface* roudiMemoryInterface) noexcept
     }
     auto introspectionMemoryManager = maybeIntrospectionMemoryManager.value();
 
+    /// @todo #415 Move the hard-coded service description to iceoryx_posh_types.hpp?
```

Comment:
Sorry, but I cant resist: https://www.youtube.com/watch?v=ZXsQAXx_ao0

Reply:
As the `iceoryx_posh_typeshpp` is already clutter. I prefer if the common info lives in `service_registry.hpp`

Reply:
Please don't do this. Yes you are right the file is cluttered but at least we have a central place for all the iceoryx settings. If you place them there we do not know what we are able to configure and what not, the only chance we then have is to look at all classes all over the code and look for settings.

Please move this to the `iceoryx_posh_types.hpp`. We already have also an issue where we would clean this file partially: #1077

Reply:
Moved to `iceoryx_posh_types.hpp` :heavy_check_mark: